README.md

```diff
@@ -1,4 +1,4 @@
-﻿# haskell-src-transformations
+# haskell-src-transformations
```

Comment:
There was some invisible character at the beginning of the file that my Markdown editor did not like and I thus removed it. I've now checked on the `main` branch. As it turns out the invisible character was a [UTF-8 Byte Order Mark `ef bb bf`](https://en.wikipedia.org/wiki/Byte_order_mark). I think removing it should be fine since a BOM is only needed for UTF-16 and UTF-32 encoded files and all other files don't have a BOM either.

Reply:
Okay, thanks for checking this.